gprofiler/gprofiler_types.py

```diff
@@ -32,6 +33,31 @@ class ProfileData:
 ProcessToProfileData = MutableMapping[int, ProfileData]
 
 
+class ProfilingErrorStack(StackToSampleCount):
+
+    PROFILING_ERROR_STACK_PTRN = re.compile(r".*;\[Profiling .+: .+\]")
```

Comment:
```suggestion
    PROFILING_ERROR_STACK_PATTERN = re.compile(r".*;\[Profiling .+: .+\]")
```

Name is long anyway, don't save on letters :sweat_smile:

Reply:
👍